fractions/netflix/hystrix/pom.xml

```diff
@@ -25,6 +25,7 @@
   <properties>
     <swarm.fraction.stability>stable</swarm.fraction.stability>
     <swarm.fraction.tags>Fault Tolerance,Netflix OSS</swarm.fraction.tags>
+    <version.hdrHistogram>2.1.9</version.hdrHistogram>
```

Comment:
Should be in the root `pom.xml`, together with a `<dependencyManagement>` entry, like all other dependecies?

Reply:
yes, but as I said, I would leave this to @mkouba to fix together with the MP TCK suite he is working on

Reply:
I was a bit eager, this is hdr dep, yes maybe you are right

Reply:
not sure, @kenfinnigan what's you opinion: where should deps like this be managed?

Reply:
Currently all netflix fraction dependencies are depMan'd in https://github.com/wildfly-swarm/wildfly-swarm/blob/master/fractions/netflix/pom.xml

Maybe there?

Reply:
+1 for netflix parent